Providers/OpenEBS/Installers/Operator/0.6/ansible/openebs.yaml

```diff
@@ -0,0 +1,36 @@
+# TODO 
+# Apply openebs operator and openebs storage class
+
+- hosts: localhost
+  connection: local  
+  tasks:
+   - name: Applying openebs operator
+     shell: kubectl --kubeconfig /root/admin.conf apply -f https://raw.githubusercontent.com/openebs/openebs/v0.6/k8s/openebs-operator.yaml 
+     args:
+       executable: /bin/bash
+
+   - name: Applying openebs storageclass 
+     shell: kubectl --kubeconfig /root/admin.conf apply -f https://raw.githubusercontent.com/openebs/openebs/v0.6/k8s/openebs-storageclasses.yaml 
+     args:
+       executable: /bin/bash
+
+   - name: Checking Maya-API-Server is running
+     shell: kubectl get po --all-namespaces | grep maya-apiserver | awk '{print $4}'
+     register: maya_api
+     until: "'Running' in maya_api.stdout"
+     delay: 30
+     retries: 100
```

Comment:
@ashishranjan738 , can we do a few further checks to ensure that the m-apiserver ports are actually working fine - mayactl version can give some good info

Reply:
Can we make use of in-built kubernetes capabilities to get status, such as `-o go-template --template "{{.status.phase}}"`